app/server/Mint/nBTCMintTxn.server.ts

```diff
@@ -0,0 +1,90 @@
+import type { Network } from "bitcoinjs-lib";
+import { opcodes, Psbt, script } from "bitcoinjs-lib";
+import Wallet from "sats-connect";
+import type { Address } from "sats-connect";
+import { fetchUTXOs, fetchValidateAddress } from "~/api/btcrpc";
+import type { UTXO, ValidateAddressI } from "~/api/btcrpc";
+import { nBTC_ADDR } from "~/lib/constant";
+import type { ToastFunction } from "~/hooks/use-toast";
+
+export async function nBTCMintTxn(
+	bitcoinAddress: Address,
+	sendAmount: number,
+	opReturnInput: string,
+	network: Network,
+	toast?: ToastFunction,
+) {
```

Comment:
Server doesn't have access to the user wallet, so this won't work.

Reply:
Done. Replied